src/backer/backering.py

```diff
@@ -296,6 +298,24 @@ def on_post(self, req, rep):
 
         cr = httping.parseCesrHttpRequest(req=req)
         serder = serdering.SerderKERI(sad=cr.payload, kind=eventing.Serials.json)
+
+        backer_identifier = serder.ked["b"]
+        # Confirm registry backer
+        if TraitCodex.RegistryBackers.value not in serder.ked["c"]:
+            raise falcon.HTTPBadRequest(falcon.HTTP_400)
+
+        contains_valid_seal = False
+        for item in serder.ked["a"]:
+            if (
+                item["bi"]
+                and item["bi"] == backer_identifier
+                and item["d"] == REGISTRAR_SEAL_SAID
+            ):
+                contains_valid_seal = True
```

Comment:
break here as we just need the first based on the spec

Reply:
updated
ref here; dab7a7ccc924fd29f3db5e45ddc13ff74f7358a7